modules/webgl/src/adapter/converters/texture-formats.ts

```diff
@@ -509,7 +509,7 @@ export function getTextureFormatSupport(
 
   // Support Check that we have a GL constant
   let supported = info.gl === undefined;
-  supported = supported && checkTextureFeature(gl, info.f as DeviceFeature, extensions);
+  supported = supported && checkTextureFeature(gl, info.f, extensions);
```

Comment:
Hard to tell in the diff, is this related?

Reply:
Huh, no, must be some some lint/prettier autofix. But if the casts are not needed, then better drop them...